bumble/avdtp.py

```diff
@@ -1397,7 +1429,19 @@ def find_remote_sink_by_codec(
                             codec_capabilities.media_type == AVDTP_AUDIO_MEDIA_TYPE
                             and codec_capabilities.media_codec_type == codec_type
                         ):
-                            has_codec = True
+                            if isinstance(
+                                codec_capabilities.media_codec_information,
+                                VendorSpecificMediaCodecInformation,
+                            ):
+                                if (
+                                    codec_capabilities.media_codec_information.vendor_id
+                                    == vendor_id
+                                    and codec_capabilities.media_codec_information.codec_id
+                                    == codec_id
+                                ):
+                                    has_codec = True
+                            else:
+                                has_codec = True
```

Comment:
nit: If there isn't additional works here
```suggestion
                            has_codec = (isinstance(
                                codec_capabilities.media_codec_information,
                                VendorSpecificMediaCodecInformation,
                            ) and codec_capabilities.media_codec_information.vendor_id == vendor_id
                            and codec_capabilities.media_codec_information.codec_id == codec_id)
```

Reply:
That's not 100% the same: if we encounter a capabilities for a standard (non-vendor) codec, we want `has_codec` to be `True`.